### ISSUE TITLE
Screen goes black when open a second game, v1.0

### ISSUE THREAD

**Issue by cesnito** [trimui/firmware_smart#1]
I bought this model and it already came with version 1.0 but when I just turn it on I open a game and then close that game, I open a second game of any emulator the screen simply goes black, I have to completely turn off the trimui, forcing it to shut down and turn it on again new

**Comment by rodrigoxxxxxxx** [comment]
> I bought this model and it already came with version 1.0 but when I just turn it on I open a game and then close that game, I open a second game of any emulator the screen simply goes black, I have to completely turn off the trimui, forcing it to shut down and turn it on again new

For me happens the same.  Every time i open a second ROM the screen goes black. I need to restart the console.

**Comment by cesnito** [comment]
> > I bought this model and it already came with version 1.0 but when I just turn it on I open a game and then close that game, I open a second game of any emulator the screen simply goes black, I have to completely turn off the trimui, forcing it to shut down and turn it on again new
> 
> For me happens the same. Every time i open a second ROM the screen goes black. I need to restart the console.

Yes, same Situation, But today It works normal, open AND clóse games without problem, i only connected the consolé to internet AND force to updated It the core AND reset retroarch config
